index.html

```diff
@@ -28,24 +28,28 @@
     <p>Hello World!</p>
 
     <section id="welcome-section">
-        <h1>Welcome to our portforlio site ;)</h1>
+        <h1>Welcome to our portforlio site </h1>
     </section>
 
     <section id='projects'>
         <div class="project-tile"></div>
-        <a href="#"></a>
+        <a href="#">Project Title</a>
     </section>
 
     <section id="contact">
         <h2>Let's work together...</h2>
-        <li>
-            <a href="facebook.com" target="_blank" class="btn contact-details">
-                <i class="fab fa-facebook-square"></i> Facebook
-            </a>
-        </li>
-        </br>
-        <li><a href="twitter.com" target="_blank" class="btn contact-details"><i class="fab fa-twitter"></i> Twitter
-        </a></li>
+        <ul>
```

Comment:
Good use of `<ul>` tags as refrenced by the previous reviewer. [https://github.com/lucy-sees/MicroverseTrials/pull/2#pullrequestreview-1473466294](url) 👍🏽